### DISCUSSION TITLE
Download PGP keys mismatch

### DISCUSSION THREAD

**Discussion by lazka** [openssl/openssl#21628] [General Discussion]
https://www.openssl.org/source/ states that

> PGP keys for the signatures are available from the [OTC page](https://www.openssl.org/community/otc.html). Current members that sign releases include Richard Levitte, Matt Caswell, Paul Dale, and Tomas Mraz.

The key used for the 3.1.2 release is not on that page and not from one of the mentioned people. Maybe this could be clarified.

**Comment by tom-cosgrove-arm** [comment]
Duplicate of #21617, and fixed by https://github.com/openssl/web/pull/415

**Reply by lazka** [reply]
Thanks!
